tests/roots/test-linkcheck-documents_exclude/conf.py

```diff
@@ -0,0 +1,5 @@
+exclude_patterns = ['_build']
+linkcheck_exclude_documents = [
+    '^broken_link$',
+    'br[0-9]ken_link',
+]
```

Comment:
Can add the trailing new line.
```suggestion
]
```